cairosvg/url.py

```diff
@@ -32,6 +32,64 @@
 URL = re.compile(r'url\((.+)\)')
 
 
+class URLFetcher(object):
```

Comment:
Guillaume,

I'm cool with keeping it simpel. The default implementation was focussed to much towards my own needs ;-). I'll put them in a subclass. No worries.

I'll update (tonight or tomorrow).

Cheers,
Erik